homeassistant/components/mysensors/sensor.py

```diff
@@ -125,12 +123,14 @@
     ),
     "V_FLOW": SensorEntityDescription(
         key="V_FLOW",
-        native_unit_of_measurement=LENGTH_METERS,
+        native_unit_of_measurement=UnitOfLength.METERS,
         icon="mdi:gauge",
+        device_class=SensorDeviceClass.DISTANCE,
```

Comment:
I don't think this is a distance. The documentation on this measurement is inconsistent. Better not to set a device class here yet.

Reply:
Damn sorry about that. 
I'll take a look on Monday

Reply:
No worries. Thanks! 👍